driver-sync/src/main/com/mongodb/client/internal/AggregateIterableImpl.java

```diff
@@ -160,6 +160,9 @@ public void toCollection() {
             if (batchSize != null) {
                 findOptions.batchSize(batchSize);
             }
+            if (allowDiskUse != null) {
```

Comment:
As above.  I don't think this is needed.

Reply:
Done